app/code/Magento/Newsletter/Model/Queue.php

```diff
@@ -202,7 +203,11 @@ public function isNew()
      */
     public function setQueueStartAtByString($startAt)
     {
-        if ($startAt === null || $startAt == '') {
+        if (!(is_string($startAt) || $startAt === null)) {
+            throw new InvalidArgumentException('Start date should be of type string or null');
```

Comment:
With `Magento\Framework\Exception\InvalidArgumentException` it should be `Phrase` 

```suggestion
            throw new InvalidArgumentException(__('Invalid start date value.'));
```